lib/src/badge_position.dart

```diff
@@ -21,25 +21,25 @@ class BadgePosition {
 
   /// Factory method that creates a new instance of this widget
   /// according to [top] and [start]
-  factory BadgePosition.topStart({double? top, double? start}) {
-    return BadgePosition(top: top ?? -5, start: start ?? -10);
+  factory BadgePosition.topStart({double top = -5, double start = -10}) {
```

Comment:
Is it intentional that `top` has a default value of `5`? It looks like all other defaults for `top` and `bottom` are `8`.